### ISSUE TITLE
[FEATURE] Adding a function to cancel the selected task progress in Kanban mode when the game is paused.

### ISSUE THREAD

**Issue by louis-hahaha** [COMP3122-2324-Project/Simpsyber#22]
Suppose that when the game is paused, we have the ability to assign staff to various tasks. However, once a task has been assigned, it cannot be cancelled. This could pose an inconvenience for users who wish to reassign staff or cancel the task. 
![截圖 2024-02-29 14 12 38](https://github.com/COMP3122-2324-Project/Simpsyber/assets/144008157/bedbe4c7-8ca9-4955-adcc-6df85e416a25)

Suggestion:
1. Add the cancel function to Kanban mode, when user click "x", it prompts the confirmation message to user. Finally cancel the task and duty off the staff.

**Comment by gzlzgzl** [comment]
Agree. In reality, changes of requirements happen all the time and sometimes it is necessary for a programmer to quit a task. Adding this feature can make the game more realistic than it is now.

**Comment by Samfung0612** [comment]
I agree with this point. When there are five employees in the same workflow, it becomes impossible to cancel a task once it has been assigned to a user. The only option currently available is to wait for the assigned employee to complete the task. Therefore, it should have a button on the memo interface to allow for the cancellation of tasks assigned to programmers.

<img width="1287" alt="螢幕截圖 2024-03-02 23 03 29" src="https://github.com/COMP3122-2324-Project/Simpsyber/assets/69749799/da422206-a1cc-4378-8f8a-90657080c14f">
